public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryBuilder.tsx

```diff
@@ -101,13 +110,22 @@ export const PromQueryBuilder = React.memo<Props>(({ datasource, query, onChange
           }
         />
       </EditorRow>
+      {showExplain && (
+        <OperationExplainedBox
+          stepNumber={1}
+          title={<RawQuery query={`${query.metric} ${promQueryModeller.renderLabels(query.labels)}`} lang={lang} />}
+        >
+          Fetch all series matching metric name and label filters.
+        </OperationExplainedBox>
+      )}
       <OperationsEditorRow>
         <OperationList<PromVisualQuery>
           queryModeller={promQueryModeller}
           datasource={datasource as DataSourceApi}
           query={query}
           onChange={onChange}
           onRunQuery={onRunQuery}
+          highlightedOp={highlightedOp}
```

Comment:
@ivanahuckova has some solution for this highlight because when hovering the delayed animation for the borders does not look great and it looks better if on hover the border change is immediate.

Reply:
I at the moment only changed the styling to be `0.5s` instead of `1s`. I think we can improve this in separate PR.

Reply:
I think the highlight is working well or I am missing something. Could you show me what's wrong?